src/main/java/de/berlin/lostberlin/model/Business/BusinessShortDao.java

```diff
@@ -0,0 +1,22 @@
+package de.berlin.lostberlin.model.Business;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class BusinessShortDao {
+    private String fName;
```

Comment:
Is it true that BusinesShortDao is a client-side representation of Business class?

It is a bit confusing, you have Business (which is a data base view), you have BusinessShort/FullDao, which, as I understand, is an (http) client level object. Why do you need DTOs at all?

As an idea: just create different packages, client/persistence where persistence contains a db wrappers, while clients are various representations of whatever clients need. Do I miss anything?

Reply:
Yes, BusinessShortDao is shown to the client when he searches for businesses by location. 

BusinessPostDto is submitted by a business during registration. BusinessUpdateDto is submitted by a business to update his account. There's no BusinessFullDao, I guess you got it confused with OrderFullDao. Anyways, I can organize the classes in packages as you suggest, if it makes it more clear. 

However, if I understand you correctly, in both cases we'll have only one class in persistence packages: Business and Order respectively. Or should DAO classes be stored in the persistence packages as well?